pkg/render/manager.go

```diff
@@ -942,25 +944,27 @@ func managerUserSpecificSettingsGroup() *v3.UISettingsGroup {
 // all of the tigera namespaces.
 //
 // Calico Enterprise only
-func managerClusterWideTigeraLayer() *v3.UISettings {
+func managerClusterWideTigeraLayer(namespace []string) *v3.UISettings {
 	namespaces := []string{
 		"tigera-compliance",
-		"tigera-dex",
 		"tigera-dpi",
 		"tigera-eck-operator",
 		"tigera-elasticsearch",
 		"tigera-fluentd",
-		"tigera-guardian",
 		"tigera-intrusion-detection",
 		"tigera-kibana",
-		"tigera-manager",
 		"tigera-operator",
 		"tigera-packetcapture",
 		"tigera-policy-recommendation",
 		"tigera-prometheus",
 		"tigera-system",
 		"calico-system",
 	}
+
+	for _, ns := range namespace {
```

Comment:
What's the behavior if there are duplicates in this slice?

Reply:
Replaced the slice into map[string]bool to avoid duplicates